dipy/workflows/align.py

```diff
@@ -704,3 +707,66 @@ def run(self, static_image_files, moving_image_files, prealign_file='',
             save_nifti(owarped_file, warped_moving, static_grid2world)
             logging.info('Saving Diffeomorphic map {0}'.format(omap_file))
             save_nifti(omap_file, mapping_data, mapping.codomain_world2grid)
+
+
+class MotionCorrectionFlow(Workflow):
+    """
+    The Motion Correction workflow allows the user to use only one type of
+    registration (such as center of mass or rigid body registration only).
```

Comment:
Not clear what this means

Reply:
Oops, I forgot to finish to update this description, good catch